mica/web/close_events.py

```diff
@@ -0,0 +1,24 @@
+from kadi import events
+from Chandra.Time import DateTime
+
+# 30 minute pad for these events
+interest_pad = 30 * 60;
+interest = [events.normal_suns,
+            events.safe_suns,
+            events.scs107s,
+            events.caps]
+
+def close_events(start, stop):
```

Comment:
This probably belongs as a utility in kadi.events.  Which of course makes development trickier, though you can toss it in the kadi web branch.